cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/Query.java

```diff
@@ -53,77 +53,71 @@
 
     /**
      * <p>
-     * Create an {@link Index} with an automatically generated name and default options.
+     * Create a JSON {@link Index} for one or more fields with an optional index name, and ensure the
+     * index is up to date.
      * </p>
      * <p>
-     * The default options are {@code indexType=JSON, tokenize=null}.
+     * Note that the sort order on fields for index definitions currently only supports the
+     * {@link com.cloudant.sync.query.FieldSort.Direction#ASCENDING} direction. If there are one or
+     * more members of {@code fieldNames} with a direction of
+     * {@link com.cloudant.sync.query.FieldSort.Direction#DESCENDING}, an exception will be thrown.
```

Comment:
Would it help to mention that query results can still be sorted in descending order?